docs/pages/getting-started.md

```diff
@@ -0,0 +1,39 @@
+# Getting started
+When you have installed all the [prerequisites](/prerequisites.md) you are good to go.
+
+Start by installing the Simple Fable template:
+```
+dotnet new --install "Fable.Template::*"
+```
+Create a directory for your project:
+```
+mkdir FableApp
+cd FableApp
+```
+Scaffold a Fable app from the template:
+```
+dotnet new Fable
+```
+This will create Fable project in this directory. Now install the npm dependencies using yarn:
+```
+yarn install
+```
+Install Fable and dotnet dependencies using Paket:
+```
+dotnet restore
+```
+At this point, when you navigate through your code using Visual Studio Code or Atom with the Ionide extension, you should have tooltips and auto-completion working. 
+
+To run the app in developement mode, you run:
+```
+dotnet fable npm-run start
+```
+This will do two things, first, it will run the Fable server/daemon in the background. Also, it will run the webpack developement server.
```

Comment:
I'd change server/daemon to just server. Maybe also link to the why page?

Another wording maybe more straight forward, stl "This will run two servers in parallel: the Fable compilation server and the webpack development server.